app/api-client.js

```diff
@@ -1,170 +1,291 @@
-const config = require('../app/config');
+const config = require('./config');
```

Comment:
Just thinking aloud, we may want to have a small `api-client` file.
And have all these classes in their own file under `apis` or even `models`, just to keep things small and sweet.

(we don't need to do it or do it now, just thinking about it)

Reply:
I agree, it needs to be split up. `APIClient` and `JWTAuth` don't belong with `Model` and `ModelSet` and the model classes should also be separate.